plugins/GSdx/GSRenderer.cpp

```diff
@@ -274,7 +287,7 @@ bool GSRenderer::Merge(int field)
 
 		if(m_regs->SMODE2.INT && m_interlace > 0)
 		{
-			if (m_interlace == 7 && m_regs->SMODE2.FFMD == 1) // Auto interlace enabled / Odd frame interlace setting
+			if(m_interlace == 7 && m_regs->SMODE2.FFMD == 1) // Auto interlace enabled / Odd frame interlace setting
```

Comment:
I agree for FFMD if it's just a 1 bit flag.
but `if (variable) basically just checks whether the variable is non-null` is not exactly true, it could use `operator bool()` (same for `==`/`!=` but with it we make our intention more explicit).
But if something is "useless" in a sense that it'd be compiled to the same code, doesn't mean it can't affect readability (and here, shorter isn't better frequently. Like with `!!foo` instead of `foo != 0`).
But yeah, it's more or less style guide question, which pcsx2 always lacked.